dev-docs/bidders/adform.md

```diff
@@ -17,7 +17,13 @@ userIds: britepoolId, criteo, id5Id, identityLink, liveIntentId, netId, parrable
 ### Bid Params
 
 {: .table .table-bordered .table-striped }
-| Name        | Scope    | Description       | Example            | Type      |
-|-------------|----------|-------------------|--------------------|-----------|
-| `mid`       | required |                   | `12345`            | `integer` |
-| `adxDomain` | optional | The Adform domain | `'adx.adform.net'` | `string`  |
+| Name        | Scope    | Description                                   | Example                    | Type      |
+|-------------|----------|-----------------------------------------------|----------------------------|-----------|
+| `mid`       | required |                                               | `12345`                    | `integer` |
+| `adxDomain` | optional | The Adform domain                             | `'adx.adform.net'`         | `string`  |
+| `priceType` | optional | An expected price type (net or gross) of bids | `'net'`                    | `string`  |
+| `mkv`       | optional | Comma-separated key-value pairs               | `'city:NY'`                | `string`  |
+| `mkw`       | optional | Comma-separated keywords                      | `'news,sport'`             | `string`  |
+| `minp`      | optional | Minimum CPM price                             | `2.55`                     | `number`  |
+| `cdims`     | optional | Comma-separated creative dimentions           | `'300x250'`                | `string`  |
```

Comment:
The validation does not allow for spaces to exist between in the CSV list, such like `300x250, 1x1`. Can you please make that clear in this doc?

Reply:
Is it the right place for such a note? https://github.com/prebid/prebid.github.io/pull/2262/commits/c6b1f06d79f2efa080f862f9b20457f5a34b384d

Reply:
I think so? @bretg does this look good to you?